### PR TITLE
Add Edge settings endpoints

### PR DESCRIPTION
**Pull request by fradinni**

### CODE REVIEW

File: src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings-update-request.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+   */
+  get sshLoginBanner(): string {
+    return this._json.ssh_login_banner;
+  }
+
+  /**
+   * Get password expiry.
+   * @returns {number}
+   */
+  get passwordExpiry(): number {
```
Comment (JoMas971) [inline src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings-update-request.ts:68]: Shouldn't be a Date ? (i'm not sure)

File: src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+   */
+  get sshLoginBanner(): string {
+    return this._json.ssh_login_banner;
+  }
+
+  /**
+   * Get password expiry.
+   * @returns {number}
+   */
+  get passwordExpiry(): number {
```
Comment (JoMas971) [inline src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings.ts:37]: Shouldn't it be a date ?

Comment (fradinni) [reply src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings.ts:37]: No it's a number of days.

Comment (JoMas971) [reply src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings.ts:37]: Ho ok perfect. I just wanted to be sure. All good to me then.

**Pull request merged**